docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,562 @@
+version: '2.4'
+services:
+  # Description: Acts as a reverse proxy for all of the sourcegraph-frontend instances
+  #
+  # Disk: none
+  # Ports exposed to other Sourcegraph services: none
+  # Ports exposed to the public internet: 7080 (HTTP) and/or 7443 (HTTPS)
+  #
+  # Note: nginx/nginx/sourcegraph_backend.conf lists all of the sourcegraph-frontend
+  # network addresses that Nginx should proxy. nginx/nginx/sourcegraph_backend.conf
+  # needs to be updated with new addresses when scaling the number of soucegraph-frontend
+  # replicas.
+  nginx:
+    container_name: nginx
+    image: 'index.docker.io/library/nginx:1.17.7@sha256:8aa7f6a9585d908a63e5e418dc5d14ae7467d2e36e1ab4f0d8f9d059a3d071ce'
+    cpus: 1
+    mem_limit: '1g'
+    volumes:
+      - '../nginx:/etc/nginx'
+    ports:
+      - '0.0.0.0:7080:7080'
```

Comment:
Add a comment here describing how someone could alter this line to listen on port 80 instead.